sw/host/provisioning/util_lib/src/lib.rs

```diff
@@ -32,3 +33,23 @@ pub fn hex_string_to_u8_arrayvec<const N: usize>(hex_str: &str) -> Result<ArrayV
         .into_iter()
         .collect::<ArrayVec<u8, N>>())
 }
+
+// Life cycle tokens are hashed using a keccak hashing algorithm. The result is
+// a 16 byte value represented as a vector of two u64s.
```

Comment:
use [doc comments](https://doc.rust-lang.org/rust-by-example/meta/doc.html#doc-comments) here
```suggestion
/// Life cycle tokens are hashed using a keccak hashing algorithm. The result is
/// a 16 byte value represented as a vector of two u64s.
```

Reply:
Modified and re-tested, PTAL